parallel_test.go

```diff
@@ -0,0 +1,91 @@
+package testcontainers
+
+import (
+	"context"
+	"testing"
+)
+
+func TestParallelContainers(t *testing.T) {
+	tests := []struct {
+		name      string
+		reqs      ParallelContainerRequest
+		resLen    int
+		expErrors int
+	}{
+		{
+			name: "running two containers (success)",
+			reqs: ParallelContainerRequest{
+				{
+					ContainerRequest: ContainerRequest{
+
+						Image: "nginx",
+						ExposedPorts: []string{
+							"10080/tcp",
+						},
+					},
+					Started: true,
+				},
+				{
+					ContainerRequest: ContainerRequest{
+
+						Image: "nginx",
+						ExposedPorts: []string{
+							"10081/tcp",
+						},
+					},
+					Started: true,
+				},
+			},
+			resLen: 2,
+		},
+		{
+			name: "running two containers (one error)",
```

Comment:
Can we add another test entry for two errors to cover maxLength of the container requests?